client/my-sites/plans-features-main/header.jsx

```diff
@@ -1,24 +1,122 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { isEmpty } from 'lodash';
 
-const PlansFeaturesMainHeader = ( { heading, subhead } ) => (
-	<div className="plans-features-main__header">
-		<h2 className="plans-features-main__heading">{ heading }</h2>
-		{ subhead && <p className="plans-features-main__subhead">{ subhead }</p> }
-	</div>
-);
+/**
+ * Internal dependencies
+ */
+import ExternalLinkWithTracking from 'components/external-link/with-tracking';
+import FormattedHeader from 'components/formatted-header';
+import { getSitePlanSlug, getSiteSlug } from 'state/sites/selectors';
+import { getSitePurchases } from 'state/purchases/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { planHasFeature } from 'lib/plans';
+import { isJetpackBackup } from 'lib/products-values';
+import {
+	JETPACK_BACKUP_PRODUCT_LANDING_PAGE_URL,
+	JETPACK_BACKUP_PRODUCTS,
+} from 'lib/products-values/constants';
+import { addQueryArgs } from 'lib/route';
+import QuerySitePlans from 'components/data/query-site-plans';
+import QuerySitePurchases from 'components/data/query-site-purchases';
+
+class PlansFeaturesMainHeader extends Component {
+	siteHasJetpackBackup() {
+		const { purchases } = this.props;
+
+		if ( isEmpty( purchases ) ) {
+			return false;
+		}
+
+		// Search through purchased products for Jetpack Backup.
+		return ! isEmpty(
+			purchases.find( purchase => purchase.active && isJetpackBackup( purchase ) )
+		);
+	}
+
+	planHasJetpackBackup() {
+		const { sitePlanSlug } = this.props;
+
+		if ( ! sitePlanSlug ) {
+			return false;
+		}
+
+		// Check if the current site plan has a backup feature.
+		return ! isEmpty(
+			JETPACK_BACKUP_PRODUCTS.find( productSlug => planHasFeature( sitePlanSlug, productSlug ) )
+		);
+	}
+
+	getSubHeader() {
+		const { siteSlug, translate } = this.props;
+		const baseCopy = translate( "Just looking for a backups? We've got you covered." );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Just looking for backups? We’ve got you covered.' )` *ES Score: 6*